trace_viewer/core/trace_model/process_memory_dump.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<!--
+Copyright (c) 2015 The Chromium Authors. All rights reserved.
+Use of this source code is governed by a BSD-style license that can be
+found in the LICENSE file.
+-->
+
+<link rel="import" href="/core/trace_model/timed_event.html">
+
+<script>
+'use strict';
+
+/**
+ * @fileoverview Provides the ProcessMemoryDump class.
+ */
+tv.exportTo('tv.c.trace_model', function() {
+  /**
+   * The ProcessMemoryDump represents a memory dump of a single process.
+   * @constructor
+   */
+  function ProcessMemoryDump(start, worldMemoryDump) {
```

Comment:
A *single* <tt>WorldMemoryDump</tt> object links to *multiple* <tt>ProcessMemoryDump</tt> objects which were taken simultaneously. Every <tt>ProcessMemoryDump</tt> points to the <tt>WorldMemoryDump</tt> it belongs to (that's the <tt>worldMemoryDump</tt> argument of the constructor above). The actual memory data will be in the process memory dumps (once we know what it will look like).